builder/generate-dmg.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+set -x
```

Comment:
The -x is there so that if the script barfs during an automated build, we'll get a trace in the installer build log rather than just "script failed with exit code foo". Lack of -e is mostly because the first iteration I wrote contained a command that could fail (harmlessly) in some circumstances.